EnglishFilter/sections/whitelist.txt

```diff
@@ -227,6 +227,8 @@ copypastecharacter.com#$##contents { visibility: visible!important; }
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/57217
 @@||local.adguard.org^
+! https://github.com/AdguardTeam/AdguardFilters/issues/61823
+@@||cdnrole.com/videojs7/video-js-*/video.min.js$domain=gounlimited.to
```

Comment:
it is not blocked

Reply:
Now I don't see this request too and the video works good